package.json

```diff
@@ -18,5 +18,9 @@
         "dev": "encore dev",
         "watch": "encore dev --watch",
         "build": "encore production --progress"
+    },
+    "dependencies": {
+        "sass": "^1.61.0",
+        "sass-loader": "^13.0.0"
```

Comment:
No quit clear why these dependencies are needed, cause no exotic css features are used so far.

Reply:
It was mostly necessary to run `yarn run encore production` on my Debian bookworm. I don't see `yarn run encore` in your Dockerfile, maybe that's why you didn't stumble upon it? (But then I wonder, why was it not necessary for you in Docker...)